### PR TITLE
about pages content updates

### PR DESCRIPTION
**Pull request by fromwindowstolinux**
updated contents for About PyCon, The PSF, and Code of Conduct

### CODE REVIEW

File: src/components/Header.astro [hunk 1]
```diff
@@ -60,7 +60,7 @@ import { SITE_TITLE } from "../consts";
               <a class="text-lg font-bold">About</a>
               <ul class="p-2">
                 <li><a href="/about" class="text-md">About PyCon</a></li>
-                <li><a href="/psf" class="text-md">PyCon PSF</a></li>
+                <li><a href="/the-psf" class="text-md">The PSF</a></li>
```
Comment (amirulabu) [inline src/components/Header.astro:63]: why the link must be "the-psf"

File: src/pages/coc.astro [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
-import Placeholder from "../components/Placeholder.astro";
+import Layout from "../layouts/BlogPost.astro";
 ---
 
-<Placeholder />
+<Layout
+    title="PyCon Code of Conduct"
+    description="PyCon Code of Conduct"
+    pubDate={new Date("January 01 2024")}
+    heroImage="/blog-placeholder-about.jpg"
```
Comment (amirulabu) [inline src/pages/coc.astro:9]: find a relevant hero image, and pub date

File: src/pages/the-psf.astro [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+import Layout from "../layouts/BlogPost.astro";
+---
+
+<Layout
+	title="About The Python Software Foundation"
+	description="About The Python Software Foundation"
+	pubDate={new Date("January 01 2024")}
+	heroImage="/blog-placeholder-about.jpg"
```
Comment (amirulabu) [inline src/pages/the-psf.astro:9]: find relevant hero image, and correct publish date

**Review by amirulabu — CHANGES_REQUESTED** [review]

File: src/components/Header.astro (hunk 1 above)
Comment (fromwindowstolinux) [reply src/components/Header.astro:63]: the content refers to The PSF US, so I just change it to that

File: src/pages/the-psf.astro (hunk 3 above)
Comment (fromwindowstolinux) [reply src/pages/the-psf.astro:9]: Can we standardized published date as 1st of Jan 2024?

File: src/pages/coc.astro (hunk 2 above)
Comment (fromwindowstolinux) [reply src/pages/coc.astro]: made banner follow theme color

**Review by amirulabu — APPROVED** [review]

**Pull request merged**